docs/extensions/controllerregistration.md

```diff
@@ -119,6 +119,19 @@ helm:
     digest: sha256:abc
 ```
 
+If needed, a pull secret can be referenced in the `ControllerDeployment.helm.ociRepository.pullSecretRef` field.
+The pull secret must be available in the `garden` namespace of the cluster where the `ControllerDeployment` is created and must contain the data key `.dockerconfigjson` with the base64-encoded Docker configuration JSON.
+It should be of type `kubernetes.io/dockerconfigjson`.
```

Comment:
Can you explain that the role label is required or enhance the example below?

Reply:
Thanks for catching!
If also added an example to `docs/deployment/setup_gardener.md`